test/specs/behaviors/Visibility/Visibility-test.js

```diff
@@ -537,4 +537,36 @@ describe('Visibility', () => {
       })
     })
   })
+
+  describe('updateOn', () => {
```

Comment:
We should add a test here that `it()` defaults to `events`.  Also, it would be great to have an event based assertion to ensure that events are still working.

Reply:
I added a new test to test the default value. All other tests work with events, so I don't think that we need to duplicate any of them.